README.md

```diff
@@ -129,6 +129,24 @@ Pipecat makes use of WebRTC VAD by default when using a WebRTC transport layer.
 pip install pipecat-ai[silero]
 ```
 
+## Running the Krisp Audio Filter
```

Comment:
I would say something like `## Reduce background noise with audio filters` (just to make it more attractive) and then a subsection for Krisp.